google-account-plugin/src/com/google/cloud/tools/intellij/stats/UsageTrackerConfigurable.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.stats;
+
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.openapi.options.SearchableConfigurable;
+
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.JComponent;
+
+/**
+ * Creates a Usage Tracking item in the settings.
+ */
+public class UsageTrackerConfigurable implements SearchableConfigurable {
+
+  private UsageTrackerPanel usageTrackerPanel;
+  private UsageTrackerManager usageTrackerManager;
+
+  public UsageTrackerConfigurable() {
+    usageTrackerManager = UsageTrackerManager.getInstance();
+  }
+
+  @NotNull
+  @Override
+  public String getId() {
+    return "google.settings.tracker";
+  }
+
+  @Nullable
+  @Override
+  public Runnable enableSearch(String option) {
```

Comment:
According to the docs:

> @return an action to perform when this configurable is opened when a search filter query is entered by the user in setting dialog. This action, for example, can select something in a tree or a list embedded in this setting page that matches the query. 

We don't need this. I'll look into possibly using a non-searchable implementation of `Configurable` so we don't have to return null